src/utils/utils.ts

```diff
@@ -484,6 +484,74 @@ export function getSourceFromLocation(source: string, location: SourceLocation):
   return [...previousLines, ...currentLine, ...followingLines].join('\n');
 }
 
+/**
+ * Given a source file and some nodes, prints them
+ * @param source solidity path to file
+ * @param locations nodes source locations
+ * @param chalkText function that highlight the nodes text locations
+ * @param surroundingLines lines surrounding highlighted lines[
+ * @returns text with highlights
+ */
+export function getSourceFromLocations(
```

Comment:
Can we use the function `getSourceFromLocation` to implement `getSourceFromLocations`, to avoid implementing the same logic twice.

We'll need to make to extract the linesAroundSource constant in `getSourceFromLocation`  into an argument and pass in a chalfText function (highlightingFunc is probably a better name) like `getSourceFomLocations`.

Reply:
Yes, you're right, actually found it easier for me to make `getSourceFromLocation` from `getSourceFromLocations`. And then realized there where very similar, so I left only the latter